squash/dashboard/views.py

```diff
@@ -71,6 +78,6 @@ class MetricsView(ListView):
 
     def get_context_data(self, **kwargs):
         context = super(MetricsView, self).get_context_data(**kwargs)
-        bokeh_script = autoload_server(None, app_path="/metrics")
+        bokeh_script = autoload_server(None, app_path="/metrics", url=bokeh_url)
```

Comment:
Sorry @jhoblitt I was careless in removing bokeh_url - though it was not necessary in this new django/bokeh integration method.